### PR TITLE
Update udp integration test with log API changes

### PR DESCRIPTION
**Pull request by MohamedSabthar**

### CODE REVIEW

File: stdlib-integration-tests/udp/tests/udp_tests.bal [hunk 1]
```diff
@@ -57,7 +57,7 @@ function testConnectionlessClient() {
 
         var sendResult = socketClient->sendDatagram(datagram);
         if (sendResult is ()) {
-            log:print("Datagram was sent to the remote host.");
+            log:printInfo("Datagram was sent to the remote host.");
```
Comment (ayeshLK) [inline stdlib-integration-tests/udp/tests/udp_tests.bal:60]: * Isn't it better to use `log:printDebug` in test-cases ?

File: stdlib-integration-tests/udp/tests/udp_tests.bal [hunk 2]
```diff
@@ -79,7 +79,7 @@ function testConnectClient() {
 
         var sendResult = socketClient->writeBytes(msg.toBytes());
         if (sendResult is ()) {
-            log:print("Data was sent to the remote host.");
+            log:printInfo("Data was sent to the remote host.");
```
Comment (ayeshLK) [inline stdlib-integration-tests/udp/tests/udp_tests.bal:82]: ditto

File: stdlib-integration-tests/udp/tests/udp_tests.bal [hunk 3]
```diff
@@ -101,13 +101,13 @@ isolated function testConnectClientReadTimeOut() {
         if (result is byte[]) {
             test:assertFail(msg = "No UDP service running on www.ballerina.io, no result should be returned");
         } else {
-            log:print(result.message());
+            log:printInfo(result.message());
```
Comment (ayeshLK) [inline stdlib-integration-tests/udp/tests/udp_tests.bal:104]: ditto

File: stdlib-integration-tests/udp/tests/udp_tests.bal (hunk 1 above)
Comment (MohamedSabthar) [reply stdlib-integration-tests/udp/tests/udp_tests.bal:60]: here we are  logging just to keep track of what happened so it's better to use `log:printInfo` IMO.

File: stdlib-integration-tests/udp/tests/udp_tests.bal (hunk 2 above)
Comment (MohamedSabthar) [reply stdlib-integration-tests/udp/tests/udp_tests.bal:82]: above-mentioned

File: stdlib-integration-tests/udp/tests/udp_tests.bal (hunk 3 above)
Comment (MohamedSabthar) [reply stdlib-integration-tests/udp/tests/udp_tests.bal:104]: above-mentioned

**Review by ayeshLK — APPROVED** [review]

**Review by Maninda — APPROVED** [review]

**Pull request merged**